### ISSUE TITLE
Text spacing audit

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#4260]
For [WCAG 1.4.12 ](https://www.w3.org/TR/WCAG22/#text-spacing)
> Text Spacing
In content implemented using markup languages that support the following text style properties, no loss of content or functionality occurs by setting all of the following and by changing no other style property: 
Line height (line spacing) to at least 1.5 times the font size;
Spacing following paragraphs to at least 2 times the font size;
Letter spacing (tracking) to at least 0.12 times the font size;
Word spacing to at least 0.16 times the font size.

We need to manually test for this functionality.  Initial tests are using the [Text Spacing Editor extension for Chrome](https://chromewebstore.google.com/detail/text-spacing-editor/amnelgbfbdlfjeaobejkfmjjnmeddaoj) and [DigitalA11y Tublets](https://www.digitala11y.com/products/tublets/).

**Comment by StephDriver** [comment]
[Clean Theme Article](https://politicalphilosophyjournal.org/article/id/15481/)

- ✅ line height of 1.5 doesn't appear to change the line height of the main text content, but does change the distance between sections, which implies there are bits of whitespace being styled there.  But this does not interfere with the information or functionality of the page.
- :x: letter-spacing override has no effect on its own, when combined with line spacing, it can be set.
- :x: word-spacing override has no effect on its own, when combined with line spacing, it can be set.
- ✅ paragraph spacing responds to the override

Other Observations
- When overrides cause the table of contents on the side to follow beyond the length of the window, it cannot be scrolled so functionality of later parts is lost.
- letter spacing and word spacing are only overriding, if the line height has already been overriden, when the line height remains at default, then the letter and word spacing options have no effect.

**Comment by StephDriver** [comment]
### Material Theme

#### [Material Article](https://www.glossa-journal.org/article/id/10082/)

:x: #4265
DOI overflows its container
✅ Otherwise, all content resizes fine

#### [Material Homepage](https://www.glossa-journal.org/)
✅  no loss of information or content

**Comment by StephDriver** [comment]
### OLH Theme

#### [OLH Article](https://www.iastatedigitalpress.com/mmb/article/id/16903/)
- ✅ no loss of content or functionality

#### [OLH Homepage](https://www.iastatedigitalpress.com/mmb/)
- ✅ no loss of content or functionality
